tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -600,6 +600,13 @@ def calibrate(self,
       raise ValueError(
           "Should specify one and only one of feed_dict_fn and input_map_fn.")
 
+    if input_map_fn:
+      for k, v in input_map_fn().items():
+        if not isinstance(k, str):
+          raise ValueError("Keys of input_map_fn must be of type str")
+        if not isinstance(v, tf.Tensor):
```

Comment:
@pooyadavoodi  in line 607, `tf` is not an imported module and the commit is breaking calibration.

Reply:
Thanks @reactivetype for pointing out, will fix shortly.

Reply:
No worries. thanks